### PR TITLE
Support for CocoaPods 1.7.0

### PR DESCRIPTION
**Pull request by dnkoutso**
@sebastianv1 @segiddins

### CODE REVIEW

File: lib/cocoapods/generate/podfile_generator.rb [hunk 1]
```diff
@@ -37,10 +37,22 @@ def podfile_for_spec(spec)
...
-                   share_schemes_for_development_pods: generator.configuration.share_schemes_for_development_pods?,
-                   warn_for_multiple_pod_sources: generator.configuration.warn_for_multiple_pod_sources?
+
+          options = {
+            deterministic_uuids: generator.configuration.deterministic_uuids?,
+            share_schemes_for_development_pods: generator.configuration.share_schemes_for_development_pods?,
+            warn_for_multiple_pod_sources: generator.configuration.warn_for_multiple_pod_sources?
+          }
+
+          if Pod::Installer::InstallationOptions.all_options.include?('generate_multiple_pod_projects')
```
Comment (dnkoutso) [inline lib/cocoapods/generate/podfile_generator.rb]: happy to hear better ways to handle this

**Comment by dnkoutso** [comment]
I have no idea why the build keeps failing I cant repro locally.

**Comment by segiddins** [comment]
Rebased, looks like it's just a couple of normal failing tests now @dnkoutso

File: lib/cocoapods/generate/podfile_generator.rb (hunk 1 above)
Comment (segiddins) [reply lib/cocoapods/generate/podfile_generator.rb]: nit: can we extract `install_options` out into a method on `generator`, so then we can just call `install! 'cocoapods', generator.install_options` ?

File: lib/cocoapods/generate/configuration.rb [hunk 2]
```diff
@@ -149,6 +149,9 @@ def ===(other)
 
       option :use_libraries, BOOLEAN, 'false', 'Whether to use libraries instead of frameworks', nil, nil, coerce_to_bool
 
+      option :generate_multiple_pod_projects, BOOLEAN, 'false', 'Whether to generate multiple Xcode projects', nil, nil, coerce_to_bool
```
Comment (segiddins) [inline lib/cocoapods/generate/configuration.rb]: please add an integration test that sets both of these options to true?

Comment (dnkoutso) [reply lib/cocoapods/generate/configuration.rb]: sure

**Pull request merged**